tests/data/expected/main/jsonschema/keep_model_order_field_references_native_deferred.py

```
@@ -0,0 +1,13 @@
+# generated by datamodel-codegen:
+#   filename:  keep_model_order_field_references.json
+#   timestamp: 2019-07-26T00:00:00+00:00
+
+from pydantic import BaseModel
+
+
+class DescriptionType(BaseModel):
+    metadata: list[Metadata] | None = None
```

Ruff (0.15.12)

[error] Undefined name `Metadata`

(F821)